src/rust/en.nana/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "nana-aidoku"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.dev]
+panic = "abort"
+
+[profile.release]
+panic = "abort"
+opt-level = "s"
+strip = true
+lto = true
+
+[dependencies]
+aidoku = { git = "https://github.com/Aidoku/aidoku-rs" }
```

Comment:
add newline for consistency

Reply:
newline between aidoku and dependencies?

Reply:
at the end of every file. this one doesn't have one